index.html

```diff
@@ -4,9 +4,58 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles/index.css" rel="stylesheet" type="text/css" />
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <main>
+        <header> <h1>
+            <div id="title">Weather Report</h1>
+            <h2>For the lovely city of ✨<span id="cityName">Seattle </span>✨</h2></div>
+        </header>
+        <div class="stats-container">
+        <section>
+            <article>
+                <h1>Temperature</h1>
```

Comment:
Style: Remember, heading tags (`h1`, `h2`, etc) should be used as titles for parts of your page to provide structure and it should make sense to pull them out and form a table of contents.